janus_core/cli/phonons.py

```diff
@@ -138,6 +127,8 @@ def phonons(
         2x2x2.
     displacement : float
         Displacement for force constants calculation, in A. Default is 0.01.
+    mesh : tuple[float, float, float]
+        Mesh for sampling. Default is (10, 10, 10).
```

Comment:
why not the same as the supercell? for the specification

Reply:
When I implemented the supercell, I don't think I realised you can do tuples, so it initially looked like it looked like `-supercell x --supercell y --supercell z`, which is how you have to pass lists, and is obviously quite messy, and led to @oerc0122's suggestion of the current form.

With a tuple, we can do `--mesh x y z`, which seems a lot neater to me.

We do lose the flexibility of a single integer alternative, but I'd probsbly lean towards changing `supercell` to match this, rather than than the other way around.

Reply:
Does the mesh want to be `float`s or `int`s?

Reply:
ints

Reply:
also is fine with me as long supercelll and mesh are consistent